docs/feature_unicode.md

```diff
@@ -97,13 +102,7 @@ You can switch the input mode at any time by using one of the following keycodes
 |`UNICODE_MODE_BSD`     |`UC_M_BS`|`UC_BSD`     |Switch to BSD input (not implemented).   |
 |`UNICODE_MODE_WINC`    |`UC_M_WC`|`UC_WINC`    |Switch to Windows input using WinCompose.|
 
-You can also switch the input mode by calling `set_unicode_input_mode(x)` in your code, where _x_ is one of the above input mode constants (e.g. `UC_LNX`). Since the function only needs to be called once, it's recommended that you do it in `eeconfig_init_user` (or a similar function). For example:
-
-```c
-void eeconfig_init_user(void) {
-  set_unicode_input_mode(UC_LNX);
-}
-```
+You can also switch the input mode by calling `set_unicode_input_mode(x)` or `set_unicode_input_mode_noeeprom(x)` in your code, where _x_ is one of the above input mode constants (e.g. `UC_LNX`). Since the first of these writes to EEPROM, you shouldn't call it unless the value of the mode has actually changed. If you want to set the value at keyboard power-on time, you can examine the value already loaded from EEPROM with `get_unicode_input_mode`, and set it with `set_unicode_input_mode_noeeprom`, from your `matrix_init_user` function.
```

Comment:
I feel this needs more clarification and/or is misleading.

What do you mean by “unless the value of the mode has actually changed”? The values of the `UC_` constants are fixed and likely won't be changing for this exact reason, so there's no need to mention it in the docs, if that's what you meant.

Using `set_unicode_input_mode_noeeprom` for changing to your preferred input mode when the board starts is kind of missing the point. This is already done by `unicode_input_mode_init` — if you want the board to start with your preferred input mode, just add `#define UNICODE_SELECTED_MODES <your mode>` to your `config.h`. You don't need to set it yourself. (Granted, this should be made clearer in the docs.)

The intended use case for `set_unicode_input_mode` is to programmatically set the input mode, e.g. from a custom keycode or other function. For example, you want `MY_KEYCODE` to switch to `UC_LNX` if you're holding Shift and `UC_WINC` if you're holding Ctrl. The difference with `_noeeprom` is that the change isn't persisted, and the board will boot up with the previous value. This paragraph should focus on that instead of explaining how to set the mode on power up.